Source/Scene/Scene.js

```diff
@@ -2282,8 +2294,8 @@ define([
             var savedCamera = Camera.clone(camera, scene._cameraVR);
 
             var near = camera.frustum.near;
-            var fo = near * 5.0;
-            var eyeSeparation = fo / 30.0;
+            var fo = near * defaultValue(scene.focalLength, 5.0);
```

Comment:
Are you sure this should be done here and not at initialization like Cesium does everywhere else?  Is it valid to set this back to `undefined`?